ManatiCR/retopais
# RP-62: Close proposals

**To do:**
- [x] Remove links to create proposals.
- [x] Change counters texts.
- [x] Remove unneded panes.

**Steps to test:**

As an anon user check the following:
- Check the the homepage doesn't have any links to create proposals.
- Check the footer also.
- Check the main menu.
- Check the Qué es Reto País page also.
